tests/mock/MockNetworkListener.h

```diff
@@ -59,7 +59,7 @@ class MockNetworkListener : public QTcpServer
     DEFLECT_API virtual ~MockNetworkListener();
 
 protected:
-    void incomingConnection(int handle) override;
+    void incomingConnection(int handle);
```

Comment:
We had a fix for that in DisplayCluster, config.h.in defines override to null, maybe we should add it to deflect too?